packages/atomic/src/components/facets/atomic-category-facet/atomic-category-facet.tsx

```diff
@@ -156,6 +160,15 @@ export class AtomicCategoryFacet
   @Prop() public injectionDepth = 1000;
   // @Prop() public customSort?: string; TODO: KIT-753 Add customSort option for facet
 
+  @FocusTarget()
+  protected showMoreFocus!: FocusTargetController;
+
+  @FocusTarget()
+  protected headerFocus!: FocusTargetController;
```

Comment:
Even for simpler use cases, such as focusing the header (which always exists), I think the utility can be helpful. I think we are better future-proofed in case some future change re-renders the header after a search is rendered.